src/mrinufft/trajectories/io.py

```diff
@@ -0,0 +1,377 @@
+"""Holds functions for reading and writing trajectories from and to binary files.
+These trajectories nees to follow the format specified as shown below:
+
+|Name          |Type |Size   |Unit   |Description                                                              |
+|--------------|-----|-------|-------|-------------------------------------------------------------------------|
+|Version       |FLOAT|1      |n.a.   |file version this new version would be “4.0”                             |
+|Dimension     |FLOAT|1      |n.a.   |2 -> 2D , 3 -> 3D                                                        |
+|FOV           |FLOAT|D      |m      |FOV size (x,y,z) : z absent if 2D dimension                              | 
+|Minimum OSF   |FLOAT|1      |n.a.   |Minimum OS for the trajectory                                            |
+|Gamma         |FLOAT|1      |Hz/T   |For Na / MRSI imaging                                                    |
+|Spokes        |FLOAT|1      |n.a.   |Number of spokes                                                         |
+|Samples       |FLOAT|1      |n.a.   |Number of samples per spoke                                              |
+|K-space center|FLOAT|1      |n.a.   |Relative value in the range [0-1] to define center of spokes             |
+|MaxGrad       |FLOAT|1      |mT/m   |Maximum absolute gradient in all 3 (or 2) directions                     |
+|recon_tag     |FLOAT|1      |n.a.   |Reconstruction tag                                    	               |
+|timestamp     |FLOAT|1      |n.a.   |Time stamp when the binary is created                                    |
+|Empty places  |FLOAT|9      |n.a.   |Yet unused : Default initialized with 0                                  |
+|kStarts       |FLOAT|D*Nc   |1/m    |K-space location start 	                                               |
+|Gradient array|FLOAT|D*Nc*Ns|unitary|Gradient trajectory expressed in the range [-1; 1] relative to MaxGrad   |
+"""
+import warnings
+import os
+from typing import Tuple, Optional
+import numpy as np
+from datetime import datetime
+from array import array
+
+
+
+def get_grads_from_kspace_points(
```

Comment:
There is some redundancy with the code in `trajectories/utils.py`, I would rather use the code already in place.

Reply:
Replaced your codes with mine cause they are better :P